md/hardware-and-software-requirements.md

```diff
@@ -39,10 +39,6 @@ Software required for Bonita BPM Platform (Bonita BPM Engine and Bonita BPM Port
 | Google Chrome | latest version (see note 5) |
 | Microsoft Edge | latest version |
 | Internet Explorer | 11.0.x |
-| **Mobile** |
-| Mozilla Firefox | latest version |
-| Google Chrome | latest version |
-| Apple Safari | latest version |
```

Comment:
This section removal has nothing to do with Java 8. Is it an error?

Reply:
it seems to be related to #631 (no link with jdk)